examples/graphql-context/graphql_context.server.out

```diff
@@ -1 +1 @@
-bal run graphql_context.bal
+$ bal run graphql_context.bal
```

Comment:
Do we need to add this `$` character at the beginning? IIRC, we removed this back then because this makes it hard to copy and paste the command to execute.
@praneesha any reason to add this back?

Reply:
That copying issue is fixed now and we need it to differentiate the command from the output in the `.out` file.